### DISCUSSION TITLE
Assembly instructions

### DISCUSSION THREAD

**Discussion by emikrish** [sn-lab/MouseGoggles#6] [Q&A] [answered]
Hi, I was hoping to build the binocular VR headset -- are there any assembly instructions for the raspberry pi components & LCD displays?  thank you!!

**Comment by misaacson01** [comment]
Hi @monrky, thanks for the question, and good timing! I do have assembly instructions, though we recently made some QoL improvements to the 3D prints and PCB which I'm validating now. I'm aiming to upload the new versions and updated assembly instructions by the end of the week, let me know if you need anything sooner.

**Reply by emikrish** [reply]
Actually, since we have already bought the previous version of the PCB, it might be beneficial to have the original assembly instructions rather than the updated one, if you still have it.

**Reply by misaacson01** [reply, marked as answer]
@monrky I Just added text for the assembly instructions. Two things to note:

1. I noticed there was one relatively insignificant part missing from the parts list which I just added: a raspberry pi socket header for the custom PCB. It's this part here: https://www.adafruit.com/product/2222, which is basically just 2x20 sockets like these: https://www.adafruit.com/product/5583, so hopefully not too inconvenient of an omission!

2. The most challenging aspect of assembling the headset is simply assembling the custom PCB: soldering the headers is beginner-friendly, but microsoldering the FPC connectors is not. I kinda glossed over it in the instructions since I know many people have their own preferred solutions for this, whether it's to go through a professional assembly service when they order PCBs, or to use a professional/semi-professional microsoldering setup they have or a nearby lab has. Let me know if you don't have a solution for this and don't have microsoldering expertise near you, but are interested in adding this skill to your repertoire, and I can start a new discussion focused on PCB assembly/microsoldering instructions that I can link to in the main instructions. I for one opted to microsolder it myself without any expensive equipment, which is tricky but definitely an option.

**Reply by emikrish** [reply]
Thank you! This is very helpful.  For the FPC connectors -- is there any chance you could share a Bill Of Materials file and pick and place list for the binocular headset PCB so that I can order it pre-assembled from JLCPCB?

**Reply by misaacson01** [reply]
That would be the ideal solution! But I believe JLCPCB will only place parts that are in their parts library. I've looked through their library (and a few other PCB printing/assembly services) and unfortunately I don't see this FPC connector on there, so I don't think this is an option right now. But I am just writing up the microsoldering instructions now which I hope will help, and if nothing else start up a discussion to see if there are any better assembly options out there.

**Reply by misaacson01** [reply]
Just posted the assembly discussion/instructions [here](https://github.com/sn-lab/mouseVRheadset/discussions/7).

**Reply by emikrish** [reply]
Thank you!  Much appreciated.    I believe JLCPCB does have the right FPC connector -- I found [this](https://jlcpcb.com/partdetail/TeConnectivity-1_23287243/C3170061) item on their directory...  let me know if this is correct and if it'd be possible to share a BOM file for it!

**Reply by misaacson01** [reply]
Can you check the link in your comment, it looks broken to me. If they do have it, I'd be happy to help make a BOM and pick-and-place file so we can try it out!

**Reply by emikrish** [reply]
Oops! So sorry -- this should work: https://jlcpcb.com/partdetail/TeConnectivity-1_23287243/C3170061

**Reply by misaacson01** [reply]
Yeah that looks right! I guess you have to order those first for them (maybe through global sourcing) and wait for it to arrive there, then you can order assembly with it? I'll work on the required assembly files, I don't have their recommended (and not free) assembly software so it might take a bit to see if it works.

**Reply by misaacson01** [reply]
I just added [BOM and Pick&Place files](https://github.com/sn-lab/mouseVRheadset/tree/main/Hardware/PCBs/Assembly%20Testing) for the headset PCB. I had to make them manually so I'm not 100% sure all of the fields are what JLCPCB is expecting, but it seems correct to me. I believe in order for this to work, you have to first order the connectors and wait for them to arrive to JLCPCB before putting in the PCB order. If you like, I can test out an assembled order first, though it'll take at least a few weeks before I know (waiting for connectors to arrive, then waiting for PCB printing/assembly/delivery).

**Reply by emikrish** [reply]
Okay! I reached out to JLCPCB and they said that the BOM file is missing "footprint" information.

**Reply by misaacson01** [reply]
It's not a standard footprint so I don't know what they want to be there; the JLCPCB part page has that field blank for the same reason. What to write... I'm thinking either something descriptive or just the manufacturer part #. I'll try the part # first (1-2328724-3), I just updated the BOM file with this. If they still complain about that then maybe I'll follow their guide to making a BOM with Altium and see if it gives anything different.

**Reply by emikrish** [reply]
it works! thank you!!   the only adjustment was naming the third column "footprint" instead of "package"

**Reply by misaacson01** [reply]
So, 3rd column name is "footprint" and the values are the part #? If so I'll update files here too. The true test will be that the assembled board actually works, only one way to find out. Are you planning to put an assembly order in?

**Reply by emikrish** [reply]
Yes, that seems to be what they wanted! We are planning on putting an assembly order in, but like you said it will probably take a while since JLCPCB doesn't have the connectors in stock.
